src/main/java/com/conveyal/r5/analyst/broker/BrokerHttpHandler.java

```diff
@@ -65,6 +66,10 @@ public void service(Request request, Response response) throws Exception {
         if (pathComponents.length < 2) {
             response.setStatus(HttpStatus.BAD_REQUEST_400);
             response.setDetailMessage("path should have at least one part");
+            TaskError error = new TaskError(null, "Path should have at least one part", "");
```

Comment:
The reason I did this is because anything that causes a 400 error is expected by the frontend to have a JSON-formatted error payload so we can display the error.